go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/containerd/go-runc v1.0.0
 	github.com/containerd/ttrpc v1.0.2
 	github.com/containerd/typeurl v1.0.2
+	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7 // indirect
```

Comment:
bolt shouldn't bring this in even indirectly afaik. Try running vendor + tidy again